src/index.html

```diff
@@ -17,6 +17,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="index.html"
+        class="logo_link"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="li">
+            <a
+              href="index.html"
+              class="nav_link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="li">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="li">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="li">
+            <a
+              href="index.html"
+              class="nav_link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="li">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="li">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="li">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="li">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              Video
+            </a>
+          </li>
```

Comment:
Avoid using class names that represent tag names like 'li'. Use a class that describes the role or meaning of the content, such as 'nav__item'.